int8 START_AT_FIRST_NODE = 0
int8 START_WHERE_ODOMETRY_LEFT_OFF = 1
int8 START_AT_GIVEN_POSE = 2

# inital_pose should be Map -> base_frame (parameter, generally base_link)
#

string filename
int8 match_type
geometry_msgs/Pose2D initial_pose
---